webserver/src/handlers/job_handler.rs

```diff
@@ -120,7 +115,8 @@ pub async fn create_job(
     user_sub: UserSub,
     pool: web::Data<DbPool>,
     job_req: web::Json<CreateJobRequest>,
-) -> Result<impl Responder, impl ResponseError> {
```

Comment:
This change break the application, revert it.

Reply:
This is addressed in the new pull request.